modules/govuk/manifests/app.pp

```diff
@@ -269,14 +269,20 @@
   if ! ($app_type in ['procfile', 'rack', 'bare']) {
     fail 'Invalid argument $app_type to govuk::app! Must be one of "procfile", "rack", or "bare"'
   }
-  if ($app_type in ['procfile', 'rack']) and ($port == 'NOTSET') {
+
+  if ($app_type in ['procfile', 'rack']) and ($port == 0) {
```

Comment:
Just realised that this conditional is redundant as any ports outside of 1025-65535 will be caught by `validate_integer` below.